scripts/calculateRevenue/protocols/beefy/index.ts

```diff
@@ -4,8 +4,12 @@ import {
   VaultInfo,
 } from './types'
 import { getStrategyContract } from '../utils/viem'
-import { RevenueResult, NetworkId } from '../../../types'
+import { TokenPriceData } from '../../../types'
 import { getVaults } from './getVaults'
+import { fetchTokenPrices } from '../utils/tokenPrices'
+import { getErc20Contract } from '../../../utils'
+
+const REVENUE_USD_PRECISION = 8
```

Comment:
why 8?

Reply:
Somewhat arbitrary, just needed good-enough precision since revenue will often be fractions of dollars. Added a comment.